source/site-administrators/engine/engine-headers-based-authentication.rst

```diff
@@ -58,16 +63,50 @@ Next, configure the URLs you require authentication by setting ``url`` to desire
 
 See :ref:`engine-site-security-guide-restrict-urls` for more information on expressions that cam be used.
 
-From the above configuration, here are the attributes that Engine expects from the headers to be provided:
+From the above configuration, here are the headers that Engine expects to be provided:
 
-- MELLON_secure_key
-- MELLON_username
-- MELLON_email
+- MELLON_secure_key (required)
```

Comment:
We should reconsider this default prefix in code and here. Once we decide on the code, we can adjust this.